k8s/upgrade/src/bin/upgrade-job/main.rs

```diff
@@ -23,6 +24,11 @@ mod upgrade;
 #[tokio::main]
 async fn main() -> Result<()> {
     init_logging();
+    info!(
```

Comment:
I think there's a mcro for this already: utils::print_package_info!();

Reply:
Let me try that one out..

Reply:
Alright using print_package_info!.